NEWS.md

```diff
@@ -1,6 +1,22 @@
-## CRE (devloping version)
+## CRE (devloping version) (2021-06-16)
```

Comment:
Only released versions have a specified date. The developing version remains without a date.